internal/pkg/template/templates/workloads/services/lb-web/cf.yml

```diff
@@ -168,6 +177,18 @@ Resources:
 {{include "alb" . | indent 2}}
 {{- end}}
 
+{{- if .ImportedALB}}
+{{- range $i, $secgrp := .ImportedALB.SecurityGroups }}
+  EnvironmentSecurityGroupIngressFromImportedALB:
```

Comment:
Does it work when we have multiple SGs? Wound't the logical ids clash 🤔

Reply:
Right...there can be up to 5, I think. Adding `{{ if ne $i 0 }}{{ $i }}{{ end }}`.